web/client/components/map/openlayers/Map.jsx

```diff
@@ -159,6 +159,9 @@ class OpenlayersMap extends React.Component {
         if (this.props.registerHooks) {
             this.registerHooks();
         }
+        // setting default text for "Text" annotations
+        this.map.setProperties({defaultTextAnnotation: this.props.mapOptions && this.props.mapOptions.defaultTextAnnotation || "New"});
```

Comment:
the map component should not know anything about annotations

Reply:
this is a duplicated comment